src/main/java/com/gizasystems/automationexercise/pages/CartPage.java

```diff
@@ -13,30 +13,81 @@ public class CartPage {
     private final By productName_h4 = By.xpath("//td[@class='cart_description']//h4");
     private final By viewCart_a = By.xpath("//div[contains(@class,'confirm')]//a[@href='/view_cart']");
 
+    private String url = System.getProperty("baseUrl") + "/view_cart";
+
+    private final By firstProductName = By.xpath("//a[normalize-space()='Blue Top']");
+
+    private final By secondProductName = By.xpath("//a[normalize-space()='Men Tshirt']");
+
+    private final By firstProductPrice = By.xpath("//td[@class='cart_price']//p[contains(text(),'Rs. 500')]");
+
+    private final By secondProductPrice = By.xpath("//td[@class='cart_price']//p[contains(text(),'Rs. 400')]");
+
+    private final By firstProductQuantity = By.xpath("//tr[@id='product-1']//button[@class='disabled'][normalize-space()='1']");
+
+    private final By secondProductQuantity = By.xpath("//tr[@id='product-2']//button[@class='disabled'][normalize-space()='1']");
+
+    private final By firstProductTotal = By.xpath("//p[@class='cart_total_price'][normalize-space()='Rs. 500']");
+
+    private final By secondProductTotal = By.xpath("//p[@class='cart_total_price'][normalize-space()='Rs. 400']");
```

Comment:
- I believe we can enhance those locators (let's discuss this next session).
- Please remove the empty lines between the locators.